### PR TITLE
Extract first paragraph

### LINKED ISSUE #164: Redesign Blog Page
# Bounty
Let's be proud of our work and feature our blog better.

## Scope

- visible first paragraph / description
- tags for blogpost
- some blog posts should be recommended
- some kind of sort possibility

## Deliverables

- updated code

## Roles
bounty gardener: @Clearwood  / 50 DAI 
bounty worker: @mwaa  / 250 DAI
bounty reviewer: name / 50 DAI

## Gardener checklist

- [x] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [ ] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

### PR DESCRIPTION
**Pull request by mwaa**
Clone on local to test the following:-

- [ ] Post sample paragraph is available on blog page
- [ ] Tags are available on right side of blog page
- [ ] Clicking on any of the tags filters the blog page content
- [ ] Update all posts to have tags

Closes #164

### CODE REVIEW

File: src/routes/blog/index.svelte [hunk 1]
```diff
@@ -2,14 +2,23 @@
...
         return { posts };
       });
   }
 </script>
 
 <script>
   import { formatDate, authors } from "./_formatters";
+  export let tags = [
+    'plasma', 'ethereum', 'scaling', 'smart contracts',
+    'update','dev', 'ecosystem', 'governance', 'voting',
```
Comment (Clearwood) [inline src/routes/blog/index.svelte:20]: maybe let's add nervos here?

**Comment by Clearwood** [comment]
Great work :+1: there is one small bug however that the smart contract tag goes out of the container depending on screen size 
image
Also it doesn't really work on mobile
image
The plasma tag for some reasons has no action associated with it.

File: src/routes/blog/index.svelte [hunk 2]
```diff
@@ -72,24 +72,30 @@
     }
   }
 
+  @media (max-width: 640px) {
+    .width-side {
+      display: none;
+    }
```
Comment (Clearwood) [inline src/routes/blog/index.svelte:78]:
```suggestion
    }
      .width-main {
      width: 100% !important;
    }
```
on mobile the blog articles should take in full screen real estate

Comment (mwaa) [reply src/routes/blog/index.svelte:78]: Done

**Comment by mwaa** [comment]
@Clearwood menu on local is due to rebuilds by svelte. Relays same menu on top of previous not sure why but on single build work fines. Does not affect the work in this scope

File: src/routes/blog/index.svelte [hunk 3]
```diff
@@ -72,10 +72,13 @@
     }
   }
 
-  @media (max-width: 640px) {
+  @media screen and (max-width: 640px) {
     .width-side {
       display: none;
     }
+    .width-main {
+      width: 100%;
```
Comment (Clearwood) [inline src/routes/blog/index.svelte:80]:
```suggestion
      width: 100% !important;
```

Comment (Clearwood) [reply src/routes/blog/index.svelte:80]: it has to have this override, otherwise it defaults to 70%

**Pull request merged**